aggregator/src/aggregation/config.rs

```diff
@@ -34,6 +34,8 @@ pub struct AggregationConfig {
     pub keccak_circuit_config: KeccakCircuitConfig<Fr>,    
     /// RLC config
     pub rlc_config: RlcConfig,
+    /// RLC config 1
+    pub rlc_config1: RlcConfig,
```

Comment:
Is it necessary to add the same config twice?

Reply:
Not necessary, it will be deleted in next commit.